.github/workflows/updateSnapshots.yaml

```
@@ -20,5 +20,20 @@ jobs:
+      - name: List artifacts from last successful workflow
+        run: |
+          curl -s -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+              -H "Accept: application/vnd.github+json" \
+              "https://api.github.com/repos/${{ github.repository }}/actions/artifacts" | jq '.artifacts[] | .name'
+
+      - name: Run Cypress tests
+        run: npm run cy:test
+
+      - name: Upload snapshots as new artifact
+        # if: always()
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)